lib/timeline.js

```diff
@@ -164,6 +192,38 @@ Timeline.prototype.unsubscribe = function (userToken, topic, callback) {
   this._request(opts, callback);
 };
 
+/**
+ * List user subscriptions
+ *
+ * @param {String} userToken
+ * @callback {callback} callback
+ */
+Timeline.prototype.listSubscriptions = function (userToken, callback) {
+  if (typeof userToken !== 'string') {
+    return process.nextTick(function () {
+      callback(new Error('Expected userToken to be a string.'));
+    });
+  }
+
+  var opts = {
+    method: 'GET',
+    endpoint: '/v1/user/subscriptions',
+    headers: {
+      'X-User-Token': userToken
+    }
+  };
+
+  this._request(opts, function (err, body, res) {
+    if (err) {
+      return callback(err);
+    }
+
+    var topics = body.topics || [];
+
+    callback(null, topics, res);
```

Comment:
The reason we're returning an array is because it makes more sense for it to return an array - which is also the reason why we're handling the body rather than returning what `_request` returns like we do in other functions. Plan is to make all API's return some useful data and not just the body and res in the next major bump.